plugins/mapper-annotated-text/src/test/java/org/elasticsearch/index/mapper/annotatedtext/AnnotatedTextFieldMapperTests.java

```diff
@@ -335,7 +335,7 @@ public void testStoreParameterDefaults() throws IOException {
         var source = source(TimeSeriesRoutingHashFieldMapper.DUMMY_ENCODED_VALUE, b -> {
             b.field("field", "1234");
             if (timeSeriesIndexMode) {
-                b.field("@timestamp", randomMillisUpToYear9999());
+                b.field("@timestamp", "2000-10-10T23:40:53.384Z");
```

Comment:
This is because now we apply default mapping for index mode when creating `MapperService` in tests, see the change in `MapperServiceTestCase`. Because that was not done previously, timestamp validation never kicked in because DataStreamTimeStampFieldMapper was not in the mapping, now it does. In real world default mapping is applied way above in `MetadataCreateIndexService`.

Reply:
I see, thanks for the explanation.